workflow/rules/qc.smk

```diff
@@ -1,41 +1,202 @@
-rule fastqc:
+rule fastqc_R1:
```

Comment:
What's the reason to separate R1 and R2 analysis in fastqc? It can handle paired end reads.

Reply:
The .zip file of FastQC is parsed by MultiQC. When reviewing the MultiQC report, it only showed the output for R1, but not for R2. I couldn't change MultiQCs behaviour to only show the one read, thus I "split" the output of FastQC.

Reply:
ok.

Reply:
Did you check this out? Somehow seems to be possible: https://github.com/ewels/MultiQC/issues/575

Reply:
ahh so it may be possible to simply delete the R1/R2 and show aggregate the files that way. I'll try this out tonight, that would be a way better solution. Moreover, tonight I'll try finally get this branch to merge. It should be easier to fix this branch since we have a way to accesses a ubuntu machine now. It was kinda hard to guess, where the test failed because the expand button Is missing (see below)

![image](https://user-images.githubusercontent.com/46334240/107536809-7c68c300-6bc2-11eb-8d7c-309171e56ea2.png)

Reply:
Yeah, from time to time, github actions still has some glitches.